### PR TITLE
[sysdig] Adding priority class option for sysdig node image analyzer

### PR DESCRIPTION
**Pull request by endriu0**
## What this PR does / why we need it:
This PR adds optional priorityclass option for image-analyzer daemonset.

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [x] PR only contains changes for one chart

### CODE REVIEW

File: charts/sysdig/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 
 This file documents all notable changes to Sysdig Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
 
+## v1.12.24
```
Comment (deepskyblue86) [inline charts/sysdig/CHANGELOG.md]:
```suggestion
## v1.12.25
```

File: charts/sysdig/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sysdig
-version: 1.12.23
+version: 1.12.24
 appVersion: 12.0.1
```
Comment (deepskyblue86) [inline charts/sysdig/Chart.yaml]:
```suggestion
version: 1.12.25
appVersion: 12.0.2
```

File: charts/sysdig/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: sysdig
-version: 1.12.23
+version: 1.12.24
```
Comment (balinaveen) [inline charts/sysdig/Chart.yaml]: @endriu0  please rebase and submit the PR. The current version is already 1.12.24 and appVersion has changed to 12.0.2.

**Review by deepskyblue86 — CHANGES_REQUESTED** [review]

File: charts/sysdig/Chart.yaml (hunk 3 above)
Comment (endriu0) [reply charts/sysdig/Chart.yaml]: Done - thanks @balinaveen

File: charts/sysdig/CHANGELOG.md (hunk 1 above)
Comment (endriu0) [reply charts/sysdig/CHANGELOG.md]: Thanks @deepskyblue86 . Completed

**Review by deepskyblue86 — APPROVED** [review]

**Pull request merged**